### PR TITLE
fix: remove tests directory from the test coverage report

### LINKED ISSUE #993: Remove tests directory from test coverage
**Describe the bug**

At present, each directory and file is included in code coverage but tests directory should not be included there
https://github.com/anitab-org/mentorship-backend/blob/8f1906eb284cc7774cc7325a4fa519afa8a45f2a/.github/workflows/main.yml#L34

**Expected behavior**

Test directory should be excluded from the coverage

**Screenshots**

qwe

### PR DESCRIPTION
**Pull request by ashwinijha6**
### Description
 Fixed the test coverage report by removing tests directory.

Fixes #993 

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

tests

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[run]
+omit = 
+    # omit everything in the tests directory
+    tests/*
+    # omit the virtual environment, don't forget to change the name of your venv in place of venv1
+    venv1/*
```
Comment (devkapilbansal) [inline .coveragerc]: I don't think that we need it. We don't make a virtualenv on Github CI while testing, therefore it can be ignored

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@ashwinijha6 instead of `bug`, type should be `fix` in your PR title, and commit message

**Comment by ashwinijha6** [comment]
> @ashwinijha6 instead of `bug`, type should be `fix` in your PR title, and commit message

Ok I will change that @devkapilbansal

File: .coveragerc (hunk 1 above)
Comment (ashwinijha6) [reply .coveragerc]: @devkapilbansal you mean to say only the virtual environment part right? If so I will remove it

Comment (devkapilbansal) [reply .coveragerc]: Yes only virtualenv

**Comment by ashwinijha6** [comment]
@devkapilbansal could you pls review my pull request I have made the required changes
Also I don't know why the checks  are failing ? seems to be the Codecov

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Comment by devkapilbansal** [comment]
> @devkapilbansal could you pls review my pull request I have made the required changes
> Also I don't know why the checks are failing ? seems to be the Codecov

You can ignore failing tests for `codecov/project`. They are because you remove tests directory from coverage

**Review by epicadk — APPROVED** [review]
👍🏼

**Comment by epicadk** [comment]
Since the changes are only to the coverage files marking it as ready to merge .

**Pull request merged**